Library/Formula/nss.rb

```diff
@@ -67,8 +65,18 @@ def install
     system "#{bin}/certutil", "-L", "-d", pwd
   end
 
+  # A very minimal nss-config for configuring firefox etc. with this nss,
+  # see https://bugzil.la/530672 for the progress of upstream inclusion.
+  def config_file; <<-EOS.undent
+    #!/bin/sh
+    for opt; do :; done
```

Comment:
With a full featured nss-config, it's possible to change its output like this:
````
~ » nss-config --prefix=/alt/prefix --cflags
-I/alt/prefix/include
````
The ones supported by this minimal version are guaranteed to be the last argument when configuring firefox/xulrunner, so I just ignore any others.